core/src/main/scala/kafka/admin/ConsumerGroupCommand.scala

```
@@ -420,27 +425,22 @@ object ConsumerGroupCommand {
                                   .ofType(classOf[String])
     val options = parser.parse(args : _*)
 
+    val useOldConsumer = options.has(zkConnectOpt)
+
     val allConsumerGroupLevelOpts: Set[OptionSpec[_]] = Set(listOpt, describeOpt, deleteOpt)
 
     def checkArgs() {
       // check required args
-      if (options.has(newConsumerOpt)) {
+      if (useOldConsumer) {
+        if (options.has(bootstrapServerOpt))
```

We should probably throw an error if --new-consumer is specified too.